contracts/experimental/ERC20ProxyForZRC2/test/basicProxyTest.ts

```diff
@@ -0,0 +1,179 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import hre from "hardhat";
+import { ScillaContract } from "hardhat-scilla-plugin";
+import { Account } from "@zilliqa-js/zilliqa";
+import { initZilliqa } from "hardhat-scilla-plugin";
+
+/// not.to.be.reverted seems to not work here (it complains about null receipts), so ..
+describe("basicTest", function () {
+  let zrc2Contract: ScillaContract;
+  let erc20Proxy: Contract;
```

Comment:
```suggestion
  let erc20Proxy: ZRC2ERC20Proxy;
```

Reply:
That's produced by typechain. Import with `import { ZRC2ERC20Proxy } from "../typechain-types/contracts/ZRC2ERC20Proxy";`